numalogic/connectors/utils/aws/exceptions.py

```diff
@@ -0,0 +1,106 @@
+class UnRecognizedAWSClientException(Exception):
```

Comment:
Create a generic exception for AWS, e.g. `AWSError(Exception)`.
And let's inherit this with everything related to it.